### DISCUSSION TITLE
Page window minimizes less than set minimum width

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#1116] [Q&A]
### Question

Hi, guys I trying to set min width for my flet app page but it's just not responding I have looked through the docs and previous questions and I am doing everything right, but flet page still won't stop you from reducing the width when you get to  the min-width here is a code snippet showing how I set the page's min-width :

ft.app(target=main)
Am I doing something wrong?
Here is a video also capturing my frustration:

![vokoscreenNG-2023-03-01_17-39-00](https://user-images.githubusercontent.com/77079200/222298324-6c5b342f-e857-40a3-8741-e33e0e408e04.gif)

### Code sample

```python
def main(page: ft.Page = ft.Page):
    page.window_min_width = 570
    page.window_min_height = 500
    page.update()

    def page_resize(e):
        pw.value = f"{page.width} px"
        pw.update()

    pw = ft.Text(bottom=50, right=50, style="displaySmall",color=ft.colors.WHITE)
    page.on_resize = page_resize
    page.overlay.append(pw)

    def updateTotalField(e: ft.ControlEvent):
        fieldOneVal = 0 if fieldOne.value == "" or any(char.isalpha() for char in fieldOne.value) or int(fieldOne.value) < 0 else int(fieldOne.value)
        fieldTwoVal = 0 if fieldTwo.value == "" or any(char.isalpha() for char in fieldTwo.value) or int(fieldTwo.value) < 0 else int(fieldTwo.value)
        fieldOneAndTwoTotal.value = str(fieldOneVal+fieldTwoVal)
        fieldOneAndTwoTotal.update()

    fieldOneAndTwoTotal = ft.TextField(
        read_only=True, width=100, label="Total")
    fieldOne = ft.TextField(
        label="One", width=100, on_submit=updateTotalField, on_blur=updateTotalField)
    fieldTwo = ft.TextField(
        label="Two", width=100, on_submit=updateTotalField, on_blur=updateTotalField)
    page.add(
        fieldOneAndTwoTotal,
        fieldOne,
        fieldTwo,
    )
```

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Could be an issue in `window_manager` package implementation: https://github.com/leanflutter/window_manager/issues/189

Wondering if setting `page.window_frameless = True` makes min size work?

**Reply by Haadi-A** [reply]
Yh could be the implementation.  I have been using window_manager in flutter, it works as expected.
